core/pyproject.toml

```diff
@@ -45,6 +45,7 @@ PyMySQL = { version="^1.0.2", optional = true }
 redis = { version = "^4.3.5", optional = true }
 SQLAlchemy = { version = "^2.0.4", optional = true }
 sqlglot = "^10.4.3"
+prometheus-client = "^0.16.0"
```

Comment:
let's keep the list alphabetically sorted.

Reply:
✅